Tracking/src/Tracking/Reco/CKFProcessor.cxx

```diff
@@ -641,7 +654,7 @@ void CKFProcessor::onProcessEnd() {
                  << std::setprecision(3) << profiling_map_["setup"] / nevents_
                  << " ms";
   ldmx_log(info) << "hits        Avg Time/Event = " << std::fixed
-                 << std::setprecision(2) << profiling_map_["hits"] / nevents_
+                 << std::setprecision(3) << profiling_map_["hits"] / nevents_
```

Comment:
We went down to precision of 2 bc depending on the machine that changes time to time, so the 3rd digit is pretty unstable anyway